pages/common/pass.md

```diff
@@ -24,6 +28,6 @@
 
 `pass generate -c {{path/to/data}} {{num}}`
 
-- Run any git command against the underlying store repository:
+- Initialize a new git repostitory (Any changes done by pass will be comitted automatically):
```

Comment:
```suggestion
- Initialize a new Git repostitory (any changes done by pass will be comitted automatically):
```
Why this example instead of the previous one? What makes it better?

Reply:
Since you only really make changes to the files using pass, and all changes made by pass are committed automatically, this is the only command most people will use.